dask_ml/metrics/classification.py

```diff
@@ -20,9 +24,8 @@ def accuracy_score(y_true, y_pred, normalize=True, sample_weight=None,
         If ``False``, return the number of correctly classified samples.
         Otherwise, return the fraction of correctly classified samples.
 
-    sample_weight : None
-        For compatibility with scikit-learn. This is currently not
-        supported.
+    sample_weight : 1d array-like, optional
+        Sample weights.
```

Comment:
Could you add a

```
    .. versionadded:: 0.7.0
```

directive below `Sample weights`.

We should start doing this for all new features / keywords.